src/webgpu/api/operation/resource_init/texture_zero.spec.ts

```diff
@@ -532,7 +532,7 @@ const checkContentsImpl: { [k in ReadMethod]: CheckContents } = {
 export const g = makeTestGroup(TextureZeroInitTest);
 
 g.test('uninitialized_texture_is_zero')
-  .params(paramsBuilder)
```

Comment:
Maybe some parts of this should be cases and some should be subcases?

Reply:
Oh mm yea I think we need some to be cases. Can you leave this one out for now? Probably need to revisit this test.